bokeh/sphinxext/bokeh_sampledata_xref.py

```diff
@@ -58,6 +58,16 @@
 # Dev API
 # -----------------------------------------------------------------------------
 
+class gallery_xrefs(nodes.General, nodes.Element):
+    pass
+
+class BokehGalleryOverviewDirective(BokehDirective):
+
+    has_content = True
```

Comment:
This should be `False`. I think this line is not needed.

Reply:
I don't quite follow. Do you mean it does not work if you set to `False`?